.github/workflows/Build.yml

```diff
@@ -8,28 +8,17 @@ jobs:
     strategy:
       matrix:
         include: 
-          - os: ubuntu-18.04
           - os: ubuntu-20.04
+          - os: ubuntu-22.04
           - os: macos-10.15
```

Comment:
Shouldn't the MacOS 10.15 also be removed from this `Build.yml` since it was removed in the other commit?

Reply:
I think you looked at outdated code. I removed that.

Reply:
Yes, sorry my bad.